vmb/matrimony/forms.py

```diff
@@ -642,6 +642,10 @@ def clean(self):
             raise forms.ValidationError(
                 "Please upload at least a single photo to proceed."
             )
+        if self.instance.photo_set.filter(primary=True).count() > 1:
```

Comment:
This does not seem to be the right place to do it.

Reply:
Any suggestions as to where it will be more appropriate?

Reply:
if I place it in views.py, profile_photos_add, like this
![image](https://user-images.githubusercontent.com/47107719/100974526-65727880-3562-11eb-82c8-a0997e3166aa.png)
it does not work.

Reply:
This should be fixed in the `model` layer itself.